bees_algorithm.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+from numpy import genfromtxt
+
+from dataclasses import dataclass, field
+from typing import List, Tuple
+
+from nutrinator.generator import Generator
+from nutrinator.generator import GeneratorConfig as GenConf
+from nutrinator.nutrionator import Nutrinator
+
+@dataclass
+class BeesConfig:
+    n: int = 10
+    m: int = 5
+    e: int = 2
+    nsp: int = 15
+    nep: int = 6
```

Comment:
This should be number of neighbours per bee, not total amount, as it would be hard for generator to guess how to divide this (ie. 3 elite bees and we want 8 solutions)